.github/workflows/update-deps.yml

```diff
@@ -11,31 +11,31 @@ on:
 
 jobs:
   cocoa:
-    uses: getsentry/github-workflows/.github/workflows/submodule-updater.yml@923a6414d7e37c07fdd743b2a6ccb40198ead7d3
```

Comment:
Do you have plans of switching to tags eventually? Instead of the sha?

Reply:
internal repo so we could, but I just didn't bother creating any yet